x-pack/plugins/observability_solution/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -362,8 +362,8 @@ export function ErrorSampleDetailTabContent({
     case ErrorTabKey.ExceptionStacktrace:
       return isPlaintextException ? (
         <PlaintextStacktrace
-          message={exceptions[0].message}
-          type={exceptions[0].type}
+          message={exceptions[0]?.message}
```

Comment:
this should not be necessary either, given the assertion on line 357 (`isPlaintextException...`)